qemu/tests/hotplug_mem.py

```diff
@@ -79,8 +80,10 @@ def start_test(self):
         if not callable(func):
             self.test.error("Unsupported memory operation '%s'" % operation)
         vm = self.env.get_vm(self.params["main_vm"])
-        vm.wait_for_login(timeout=login_timeout)
+        session = vm.wait_for_login(timeout=login_timeout)
         bootup_time = time.time() - vm.start_time
+        arg = "movable_node"
+        utils_test.update_boot_option(vm, args_added=arg)
```

Comment:
LGTM, Just a tiny suggestion, since the `arg` is only needed here one time to pass to the function, I suppose it is unnecessary to allocate an argument for it. No strong option, it's up to you.

Reply:
@vivianQizhu Yes, you are right.  Updated, thanks.